src/components/header/Header.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {Link} from "react-router-dom";
+
+
+const Header = ({hrefTwo,nameOne,nameTwo,nameThree, imgClassName,innerClassName,outerClassName,headerClassName,divClassName,src,alt,name,to,target,href,}) => {
+    return (
+        <header className={headerClassName}>
+
+            <div className={divClassName}>
+
+                <nav className={outerClassName}>
+
+                    <ul className={innerClassName}>
+                        <li>
+                            <Link
+                            to={to}
+                            >
```

Comment:
Wanneer je niet veel properties meegeeft, dan vind ik het altijd wat beter te lezen als het op één regel staat
```suggestion
                            <Link to={to}>
```